src/main/java/objects/Chunk.java

```diff
@@ -128,18 +128,18 @@ public void addItemsToDB() {
         }
         //objects.Mob Drops
         for (Mob mob : mobs) {
-            ArrayList<String> drops = mob.getDrops();
-            for (String drop : drops) {
-                UnlockedItemDatabase.addToDB(ItemDatabase.getItem(drop));
+            Item[] drops = mob.getDrops();
```

Comment:
Mobs will not contain the item object, just the name of the items they drop.
Limitation due to the format the data will be coming in as.